src/helpers/window.js

```diff
@@ -11,7 +11,7 @@ export default (name, options) => {
   const stateStoreFile = `window-state-${name}.json`;
   const defaultSize = {
     width: options.width,
-    height: options.height,
+    height: options.height
   };
```

Comment:
Trailing commas in objects were introduced in ECMAScript 5, and seem to be the style of this file 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas